fjord/__init__.py

```diff
@@ -1,7 +1,16 @@
+# First we import absolute imports from the future, so that our celery.py
+# module will not clash with the library
+from __future__ import absolute_import
+
 import os
 import site
 import sys
 
+from fjord.manage_utils import monkeypatch
```

Comment:
I agree we want to switch to this, but not until after we ditch `vendor/`. So I vote we keep things simple for now and not do the monkeypatch thing here.